docs/_admin/index.md

```diff
@@ -1,8 +1,4 @@
----
-layout: page
-title: Project Administration
-order: -1
----
+# Project Administration
```

Comment:
This file has a table, which it seems recommonmark doesn't support.
https://github.com/rtfd/recommonmark/issues/3
https://talk.commonmark.org/t/tables-in-pure-markdown/81/121

Reply:
Using eval_rst and an rST table ought to work
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#tables

Reply:
Fixed in 51c7edb9bd58b5378dfc66bf1bc329a0c6cec06c

Reply:
Nice catch! Thanks